reverse-proxy/nginx-tls.conf

```
@@ -0,0 +1,26 @@
+# <server_name must be replaced with the actuall server name>
+#
+# If installed, the snakeoil certificate is an automatically generated self
+# signed certificate. For a proper installation a proper certificate like one
+# from https://letsencrypt.org/ is needed.
+
+server {
+    listen 80;
+    listen [::]:80;
+    server_name <server_name>;
```

You can just let Nginx grab all of the names. This also means that people don't need to adjust this:

```suggestion
    server_name _;
```